tests/dryrun_test.go

```diff
@@ -61,7 +61,8 @@ var _ = Describe("[sig-compute]Dry-Run requests", func() {
 		resource := "virtualmachineinstances"
 
 		BeforeEach(func() {
-			vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			vmi = libvmi.NewAlpine()
+			vmi.ObjectMeta.Namespace = util.NamespaceTestDefault
```

Comment:
This should be not required. Can you please file a bug/issue?

Reply:
I wondered if someone would call me out about this. With your permission, I'll take these files' cleanup to a follow up PR.

Reply:
I know you are not introducing anything new but please consider including this change in this PR.
It should be a simple replacement, for example, replace all `vmi.Namespace` with `util.NamespaceTestDefault` and so on.